src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM = new Random();
```

Comment:
Don't use static methods or static variables in your solution. Instead, you can instantiate the `Random` class inside the method where it's needed or as an instance variable.